File: Assignment_25/program1.c
////////////////////////////////////////////////////////////////////
//
// Name :          program1.c
// Discription :   String Convert into lower case
// Input :         String
// Output :        String
// Author :        Shivam Rajendra Kale
// Date :          08/06/2025
//
////////////////////////////////////////////////////////////////////

/*
  Start
    Accept String from user 
      String Convert into lower case
  Stop    
*/

#include<stdio.h>

void strlwrx(char *str)
{

   if(NULL == str)
   {
      return ;
   }

   while(*str != '\0')
   {
      if((*str >= 'A') && (*str <= 'Z'))
      {
         *str = *str +32;   
      }
      printf("%c",*str);
      str++;
   }

}

int main()
{
   char Arr[20] = {'\0'};

   printf("Enter string :\n");
   scanf("%[^'\n']s",Arr);

   strlwrx(Arr); 

   return 0;

}

File: Assignment_54/program1.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program1.java
// Discription :   use to Reverse the digits of all elements 
// Input :         Integer
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          18/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class MyArray
{
    public void RevarseArray(int Arr[])
    {
        int i = 0,iDigit = 0, iReverse = 0;
        
        for(i = 0; i < Arr.length; i++)
        {
            while(Arr[i] != 0)
            {
                iDigit = Arr[i] % 10;
                Arr[i] = Arr[i] / 10;
                iReverse = iReverse * 10 + iDigit;
            }
            
            Arr[i] = iReverse;
            iReverse = 0;
        }

        for(i = 0; i < Arr.length; i++)
        {
           System.out.print(Arr[i]+"\t");
        }
        
    }
}

class program1
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int i = 0;

        System.out.println("Enter isze of first Array : ");
        int iSize = sobj.nextInt();

        int Arr[] = new int [iSize];

        System.out.println("Enter elements : ");
        for(i = 0; i < iSize; i++)
        {
            Arr[i] = sobj.nextInt();
        }

        MyArray mobj = new MyArray();

        mobj.RevarseArray(Arr);

        sobj.close();
    }
}